xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/displayers/BaseDisplayer.vue

```diff
@@ -27,64 +27,97 @@
-      <slot name="viewer">
-        <!--
-          Default Viewer widget
-          Normally this should rarely be used, as the default displayer
-          should provide a default viewer if no displayer is specified
-          However, this is useful if a custom displayer only implement
-          its Editor widget, as a default Viewer widget would still be provided
-        -->
-        <span>{{ value }}</span>
-      </slot>
-      <span v-if="!isViewable" v-html="$t('livedata.displayer.emptyValue')"></span>
-    </div>
+  <!-- 
+  Deactivate the popover on non editable entries by setting the trigger configuration to manual.
+  Since we don't do anything manually, that make the popover disabled.
+  -->
+  <TippyComponent
+    class="displayer-actions-popover"
+    interactive
+    :trigger="isEditable && !duringEditing ? 'mouseenter focus' : 'manual'"
+    theme="light-border"
+    placement="bottom"
+    arrow
+    ref="tippy">
+    <template #trigger>
 
-    <!-- The slot containing the displayer Editor widget -->
-    <div @keypress.enter="applyEdit"
-         @keydown.esc="cancelEdit"
-         v-if="!isView && !isLoading"
-         tabindex="0"
-         ref="editBlock"
-    >
-      <slot name="editor">
+      <div
+        :class="{ view: isView, edit: !isView, editable: isEditable }"
+        ref="displayerRoot"
+        @mouseover="showPopover = true"
+        @mouseout="showPopover = false"
+        @keypress.self.enter="setEdit"
+        v-touch:tap="touchHandler"
+      >
         <!--
-          Default Editor widget
-          Normally this should rarely be used, as the default displayer
-          should provide a default editor if no displayer is specified
-          However, this is useful if a custom displayer only implement
-          its Viewer widget, as a default Editor widget would still be provided
+          The base displayer contains three slots: `viewer`, `editor`, and `loading`.
+          It displays `viewer` or `loading` according to its current state: `this.isView` when `this.isLoading` is
+          false, and `loading` otherwise.
         -->
-        <input
-          class="default-input"
-          type="text"
-          size="1"
-          v-autofocus
-          v-model="baseValue"
-        />
-      </slot>
+
+        <!-- The slot containing the displayer Viewer widget -->
+        <div v-if="isView && !isLoading">
+          <slot name="viewer">
+            <!--
+              Default Viewer widget
+              Normally this should rarely be used, as the default displayer
+              should provide a default viewer if no displayer is specified
+              However, this is useful if a custom displayer only implement
+              its Editor widget, as a default Viewer widget would still be provided
+            -->
+            <span>{{ value }}</span>
+          </slot>
+        <span v-if="!isViewable" v-html="$t('livedata.displayer.emptyValue')"></span>
```

Comment:
That's odd of the formatting, I usually fully format my file and have good results (just tried and it fixed the formatting error).
I use `v-html` because the translation value contains a html tag

Reply:
> I use v-html because the translation value contains a html tag

Translation values should not contain HTML. We shouldn't rely on the person doing the translation to write proper HTML.